app/post_metadata.go

```diff
@@ -74,7 +74,7 @@ func (a *App) PreparePostForClient(originalPost *model.Post, isNewPost bool) *mo
 	}
 
 	// Files
-	if fileInfos, err := a.getFileMetadataForPost(post); err != nil {
+	if fileInfos, err := a.getFileMetadataForPost(post, isNewPost); err != nil {
```

Comment:
Is the rationale for not exclusively reading from the `master` on edit because you can't retroactively attach files to posts?

Reply:
for now

Reply:
Yes, I think we can set it here always to true, let me think about it, anyway, create and edit messages with attachments is not so common anyway.

Reply:
It is not so simple, I going to make some extra changes.

Reply:
Done as part of the new independent commit. @enahum @lieut-data PTAL.